docs/SceneWalkthrough/CustomInvitesWalkthrough.md

```diff
@@ -0,0 +1,10 @@
+## **Custom Invites Demo**
+This demo showcases the custom invite functionality allowing for messaging with arbitrary payloads. This is meant for cases where an already existing invitation system exists. This demo allows users to send invites with a text payload to anyone on their friends list.
+- The ``Payload`` field is where the text payload should be input.
+- The ``Set`` button finalizes the invitation creation allowing for it to be sent.
+- The ``Invite`` button next to any name on the friends list sends the currently set invitation to that user.
```

Comment:
a more definitive note that a payload must be set for the invite button to look/be enabled would be nice
maybe a comment on then needing to wait or mash the refresh button if its still isnt enabled